sdk/cli/src/commands/platform/application/services.ts

```diff
@@ -73,7 +73,12 @@ export function servicesCommand() {
       "The application unique name to list the services in (defaults to application from env)",
     )
     .addOption(new Option("-t, --type <type...>", "The type(s) of service to list").choices(SERVICE_TYPES))
-    .addOption(new Option("-o, --output <output>", "The output format").choices(["wide", "json", "yaml"]))
+    .addOption(
+      new Option(
+        "-o, --output <output>",
+        "The output format. For 'wide' format, disable terminal line wrapping for best results (use 'tput rmam' command on Unix systems)",
```

Comment:
can't we just disable it and enable it again after the command was executed? because how i understand it it will just truncate and print whatever it can show right? same as `kubectl get nodes -o wide`?

Reply:
<img width="975" alt="image" src="https://github.com/user-attachments/assets/66f3e5a3-1f73-414e-b10c-78ac9aa757f2" />

i resized my screen after the first get

Reply:
For me the 'kubectl get node -o wide' also does wrapping. It all depends on the settings of the terminal, I removed the comment. using the zoom of the terminal is also an option to make everything fit on screen